dqc/system/mol.py

```diff
@@ -294,6 +295,36 @@ def getparamnames(self, methodname: str, prefix: str = "") -> List[str]:
         else:
             raise KeyError("Unknown methodname: %s" % methodname)
 
+    def make_copy(self, **kwargs) -> Mol:
+        """
+        Returns a copy of the system identical to the orginal expect for new
+        parameters set in the kwargs.
+
+        Arguments
+        ---------
+        kwargs
```

Comment:
Please use `**kwargs` here instead of `kwargs` to differentiate between `method(kwargs)` and `method(**kwargs)`

Reply:
done